src/test/java/org/mskcc/cmo/metadb/ValidRequestCheckerTest.java

```diff
@@ -0,0 +1,122 @@
+        Assert.assertNotNull(modifiedRequestJson);
+    }
+
+    /**
+     * Test for handling all null or empty fields in requestJson
+     */
+    @Test
+    public void testNullJsonFieldHandlingInRequestJson() throws Exception {
+        MockJsonTestData requestJson = mockedRequestJsonDataMap.get("mockRequest4JsonNullOrEmptyValues");
+        String modifiedRequestJson = validRequestChecker.getFilteredValidRequestJson(requestJson.getJsonString());
+        Assert.assertNull(modifiedRequestJson);
+    }
+    
+    /**
+     * Test for handling null or empty sampleMetadata in requestJson
+     */
+    @Test
+    public void testSingleNullSampleManifestInRequestJson() throws Exception {
+        MockJsonTestData requestJson = mockedRequestJsonDataMap.get("mockRequest2aEmptySampleManifestValues");
+        String modifiedRequestJson = validRequestChecker.getFilteredValidRequestJson(requestJson.getJsonString());
+        Assert.assertNull(modifiedRequestJson);
+    }
+    
+    
+    /**
+     * Test for handling request with no valid samples
+     */
+    @Test
+    public void testRequestJsonWithNoValidSamples() throws Exception {
+        MockJsonTestData requestJson = mockedRequestJsonDataMap.get("mockRequest1cJsonDataWithMAS");
+        String modifiedRequestJson = validRequestChecker.getFilteredValidRequestJson(requestJson.getJsonString());
+        Assert.assertNull(modifiedRequestJson);
+    }
+    
+    /**
+     * Test for handling request with 2 valid samples and 2 invalid samples
+     * 
+     */
+    @Test
+    public void testRequestJsonWithTwoInvalidSamples() throws Exception {
+        MockJsonTestData requestJson = mockedRequestJsonDataMap.get("mockRequest1dJsonDataWithMFS");
+        String modifiedRequestJson = validRequestChecker.getFilteredValidRequestJson(requestJson.getJsonString());
+        Assert.assertNotSame(modifiedRequestJson, requestJson.getJsonString());
+    }
+    
+    /**
+     * Test for handling request with available fields to determine sample type abbreviation
+     * 
+     */
+    @Test
+    public void testRequestJsonWithMissingFieldsUsedForSTA() throws Exception {
+        MockJsonTestData requestJson = mockedRequestJsonDataMap.get("mockRequest1aJsonDataWithSTA");
+        String modifiedRequestJson = validRequestChecker.getFilteredValidRequestJson(requestJson.getJsonString());
+        Assert.assertNotNull(modifiedRequestJson);
+    }
+    
+    /**
+     * Test for handling request with available fields to determine nucleic acid abbreviation
+     * 
+     */
+    @Test
+    public void testRequestJsonWithMissingFieldsUsedForNAA() throws Exception {
+        MockJsonTestData requestJson = mockedRequestJsonDataMap.get("mockRequest1bJsonDataWithNAA");
+        String modifiedRequestJson = validRequestChecker.getFilteredValidRequestJson(requestJson.getJsonString());
+        Assert.assertNotNull(modifiedRequestJson);
+    }
+    
+    private String getErrorMessage(Map<String, String> errorsMap) {
```

Comment:
doesnt look like this method is actually being used so if we dont need it then no need to keep it either